test/act-mapping/preprocessor.js

```diff
@@ -0,0 +1,47 @@
+var path = require('path');
+var fs = require('fs');
+var template = fs.readFileSync(path.resolve(__dirname, 'runner.js'), 'utf-8');
+
+var actRepoDir = 'node_modules/act-rules.github.io/';
+var testcaseFilePath = path.resolve(actRepoDir, 'testcases.json');
+var testcaseContent = require(testcaseFilePath);
+var testcases = testcaseContent.testcases;
+
+createActPreprocessor.$inject = ['logger'];
+// Simple Karma preprocessor, takes JSON, outputs a JS file that can run tests
+function createActPreprocessor(logger) {
+  var log = logger.create('preprocessor.act');
+
+  return function(actRuleJson, file, done) {
+    try {
+      log.debug('Processing "%s".', file.originalPath);
```

Comment:
This should update the name to be a `.js` file. Right now it saves it as a `.json` file and karma warns for all files saying it defaults to `.js` (thus why it still works).

> 01 02 2021 09:09:10.033:WARN [middleware:karma]: Unable to determine file type from the file extension, defaulting to js.
  To silence the warning specify a valid type for /Users/stevenlambert/deque/axe-core/test/act-mapping/form-field-acc-name.json in the configuration file.
  See http://karma-runner.github.io/latest/config/files.html

```js
file.path = file.originalPath.replace(/\.json$/, '.js');
```